examples/example1.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python3
+
+'''
+This example generates the transition from /a/ to /i/ using the vocal tract
+model and the function vtlSynthesisAddTract(...).
+
+.. note::
+
+    This example uses ``ctypes`` and is very close to the vocaltractlab API.
+    This comes with breaking with some standard assumptions one have in python.
+
+If you are not aware of ``ctypes`` read the following introduction
+https://docs.python.org/3/library/ctypes.html
+
+For an in-depth API description look at the `../include/VocalTractLabApi/VocalTractLabApi.h`.
+
+For plotting and saving results you need to install ``matplotlib``, ``numpy``,
+and ``scipy``.
+
+'''
+
+import ctypes
+import sys
+
+# try to load some non-essential packages
+try:
+    import numpy as np
+except ImportError:
+    np = None
+try:
+    from scipy.io import wavfile
+except ImportError:
+    wavefile = None
+try:
+    import matplotlib.pyplot as plt
+except ImportError:
+    plt = None
+
+
+# load vocaltractlab binary
+_FILE_ENDING = ''
+if sys.platform.startswith('linux'):
+    _FILE_ENDING = '.so'
+elif sys.platform.startswith('win32'):
+    _FILE_ENDING = '.dll'
+elif sys.platform.startswith('darwin'):
+    _FILE_ENDING = '.dylib'
+
+VTL = ctypes.cdll.LoadLibrary('../lib/libVocalTractLabApi' + _FILE_ENDING)
```

Comment:
To clarify the paths:

```
'../lib/libVocalTractLabApi.so'  # Linux
'../lib/Release/VocalTractLabApi.dll'  # Windows
'../lib/libVocalTractLabApi.dylib'  # Mac??
```

If the library should be put into `../lib/Release/VocalTractLabApi.[so|dll|dynlib]` the cmake code needs to be adjusted accordingly.

Reply:
Interesting, that seems to be build system specific. On Windows, if CMake uses MSBuild, the binaries are placed in `lib/{config}/*`.

You seem to be working on Linux. Would you mind changing the CMakeLists so that GCC also puts the binaries in the same folder?

Reply:
I have never used cmake before and somehow don't get it to work. The following tutorial was close enough, so that I could change the target dir for a fixed path, but I neither get rid of the "lib" in front of VocalTractLabApi.so neither can I specify the path to be dependent on the config.

https://cmake.org/cmake/help/latest/guide/tutorial/Selecting%20Static%20or%20Shared%20Libraries.html

and

https://cmake.org/cmake/help/latest/command/add_library.html

> Adds a library target called /name/ to be built from the source files listed in the command invocation. The /name/ corresponds to the logical target name and must be globally unique within a project. The actual file name of the library built is constructed based on conventions of the native platform (such as lib/name/.a or /name/.lib).